### PR TITLE
PR: users CRUD completed

### PR DESCRIPTION
**Pull request by StovMedina**
complete CRUD for users

### CODE REVIEW

File: src/lib/db.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const config = require("./config");
 const connect = () => {
   return new Promise(async (resolve, reject) => {
     const { user, password, host } = config.db;
+    mongoose.set("strictQuery", false);
```
Comment (emiliodeleonhdez) [inline src/lib/db.js:7]: Duplicated line, please verify

File: src/models/users.js [hunk 2]
```diff
@@ -3,20 +3,15 @@ const { Schema } = mongoose;
...
-  clients: {},
-  products: {},
-  dashboard: {},
+  password: { type: String, required: true, trim: true, minlength: 8}, 
+  shoppingHistory: {type: Array}
 });
 
-const model = mongoose.model("userAdministrator", schema);
+module.exports= mongoose.model("users", schema);
+// const model = mongoose.model("users", schema);
```
Comment (emiliodeleonhdez) [inline src/models/users.js]: Remove comment lines

File: src/routes/indexRouter.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-const productsRouter = require("./productsRouter");
+// const productsRouter = require("./productsRouter");
```
Comment (emiliodeleonhdez) [inline src/routes/indexRouter.js]: Remove comment line

File: src/routes/usersRouter.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const router = express.Router();
+const user = require("../usecases/users");
+
+router.post("/create", async (req, res) => {
```
Comment (emiliodeleonhdez) [inline src/routes/usersRouter.js]: maybe change /create to /post

**Review by emiliodeleonhdez — CHANGES_REQUESTED** [review]
Resolve comments before merge

File: src/lib/db.js (hunk 1 above)
Comment (StovMedina) [reply src/lib/db.js:7]: Done.

File: src/routes/indexRouter.js (hunk 3 above)
Comment (StovMedina) [reply src/routes/indexRouter.js]: there was an error about the commented line, we can't erase it, just uncomment it.

File: src/models/users.js (hunk 2 above)
Comment (StovMedina) [reply src/models/users.js]: Done!

File: src/routes/usersRouter.js (hunk 4 above)
Comment (StovMedina) [reply src/routes/usersRouter.js]: Done!

**Review by emiliodeleonhdez — APPROVED** [review]
Go ahead!

**Pull request merged**